src/frontend/src/common/components/library/data_subview/components/EditSamplesConfirmationModal/index.tsx

```diff
@@ -82,8 +84,19 @@ const EditSamplesConfirmationModal = ({
   };
 
   const handleClose = function () {
+    setLoseProgressModalOpen(true);
+  };
+
+  const handleCloseAfterConfirmation = function () {
+    setLoseProgressModalOpen(true);
```

Comment:
why do you set the modal open to true, and then later set it to false in the same function?